docs/src/app/components/pages/components/SelectField/ExampleCustomLabel.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import SelectField from 'material-ui/lib/SelectField';
+import MenuItem from 'material-ui/lib/menus/menu-item';
+
+export default class SelectFieldCustomLabelExample extends React.Component {
```

Comment:
> I don't like to see some follow and some not. 

I agree. I think that the first component was right: 
https://github.com/callemall/material-ui/blob/master/docs/src/app/components/pages/components/AppBar/ExampleIconMenu.jsx